app/views/catalog/_home_text.html.erb

```diff
@@ -33,7 +33,7 @@
     <%= link_to "https://library.princeton.edu/about/languagestatement", target: "_blank", title: "Opens in a new tab" do %>
       <h3>Statement on language in description</h3>
     <% end %>
-    <p>Princeton University Library aims to describe library materials in a manner that is respectful to the individuals and communities who create, use, and are represented in the collections we manage. <a href="https://library.princeton.edu/about/languagestatement" target="_blank" title="Opens in a new tab">Read more...</a></p>
+    <p>Princeton University Library aims to describe library materials in a manner that is respectful to the individuals and communities who create, use, and are represented in the collections we manage. <a href="https://library.princeton.edu/about/languagestatement" target="_blank" title="Opens in a new tab" aria-label="Full Statement on Language in Description">Read more...</a></p>
```

Comment:
Can we update the aria-label to: `aria-label="Read more about full Statement on Language in Description"`